app/controllers/audits/allocations_controller.rb

```diff
@@ -28,7 +28,7 @@ def content_ids
       if params[:select_all_pages]
         FindContent.all(build_filter).pluck(:content_id)
       else
-        params[:content_ids]
+        params.fetch(:content_ids) { [] }
```

Comment:
How about passing in a default instead of supplying a block?

```rb
params.fetch(:content_ids, [])
```

Reply:
Addressed in https://github.com/alphagov/content-performance-manager/pull/315/commits/8c412454cbfb3a9b509dc5a23a543cc59afae50f